src/harness/projectsRunner.ts

```diff
@@ -422,10 +422,16 @@ class ProjectRunner extends RunnerBase {
         }
 
         function getErrorsBaseline(compilerResult: CompileProjectFilesResult) {
-            const inputFiles = ts.map(compilerResult.configFileSourceFiles.concat(
-                compilerResult.program ?
-                    ts.filter(compilerResult.program.getSourceFiles(), sourceFile => !Harness.isDefaultLibraryFile(sourceFile.fileName)) :
-                    []),
+            const inputSourceFiles = compilerResult.configFileSourceFiles.slice();

```

Comment:
OK, this one was mostly just because I didn't want to see a closure in a `filter` in a `?:` in a `concat` in a `map` and wanted to break it up. But also because `configFileSourceFiles` is now readonly and #17076 caused a compile error.